sdk/core/Azure.Core/tests/samples/Sample1_HelloWorld.cs

```diff
@@ -1,36 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for
-// license information.
-
-using Azure.Core.Pipeline;
-using NUnit.Framework;
-using System;
-using System.IO;
-using System.Threading.Tasks;
-
-namespace Azure.Core.Samples
```

Comment:
Why was this removed?

Reply:
That wasn't intended. I will revert that.